### ISSUE TITLE
Issue with the "Settings" page

### ISSUE THREAD

**Issue by fabriziobagala** [uBlockOrigin/uBlock-issues#1440]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [x] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [x] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

Blank settings page. This issue occurs only once and immediately after installation.

### A specific URL where the issue occurs

chrome-extension://.../dashboard.html#settings.html

### Steps to Reproduce

1. Open Google Chrome.
2. Install the uBlock origin extension.
3. Click on the uBlock Origin icon in the toolbar.
4. In the extension window, click on the "Open Dashboard" button.

### Expected behavior:

Show the settings page.

### Actual behavior:

Blank settings page with only the menu.

### Your environment

* uBlock Origin version: 1.31.2
* Browser Name and version: Google Chrome 87.0.4280.88 (Official Build) (x86_64)
* Operating System and version: macOS Catalina 10.15.7

**Comment by gorhill** [comment]
I can't reproduce, if nobody is able to reproduce within a week, I will close the issue as invalid. I expect that if someone is able to reproduce has developer skills, they will investigate the browser console to find out hints of what is happening.

**Comment by krystian3w** [comment]
Maybe needed is macOS and latest ARM M1 Silicon processor.

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/1193 ?
